cmd/lotus-storage-miner/info.go

```diff
@@ -218,28 +218,28 @@ func infoCmdAct(cctx *cli.Context) error {
 	}
 	spendable = big.Add(spendable, availBalance)
 
-	fmt.Printf("Miner Balance:    %s\n", color.YellowString("%s", types.FIL(mact.Balance).Short()))
-	fmt.Printf("      PreCommit:  %s\n", types.FIL(lockedFunds.PreCommitDeposits).Short())
-	fmt.Printf("      Pledge:     %s\n", types.FIL(lockedFunds.InitialPledgeRequirement).Short())
-	fmt.Printf("      Vesting:    %s\n", types.FIL(lockedFunds.VestingFunds).Short())
-	color.Green("      Available:  %s", types.FIL(availBalance).Short())
+	colorTokenAmount("Miner Balance:    %s\n", mact.Balance)
+	colorTokenAmount("      PreCommit:  %s\n", lockedFunds.PreCommitDeposits)
+	colorTokenAmount("      Pledge:     %s\n", lockedFunds.InitialPledgeRequirement)
+	colorTokenAmount("      Vesting:    %s\n", lockedFunds.VestingFunds)
```

Comment:
Not all of those had (the same) color because we wanted to highlight the actually important values with it. I'd apply this only to `Available`/`Spendable` amounts